backend/de.metas.contracts/src/main/java/de/metas/contracts/subscription/callout/C_OrderLine.java

```diff
@@ -79,6 +79,7 @@ public void onFlatrateConditions(final I_C_OrderLine ol, final ICalloutField fie
 		{
 			final BigDecimal qtyOrdered = orderLineBL.convertQtyEnteredToStockUOM(ol).toBigDecimal();
 			ol.setQtyOrdered(qtyOrdered);
+			ol.setQtyEnteredInPriceUOM(qtyOrdered);
```

Comment:
```suggestion
			final BigDecimal qtyEnteredInPriceUOM = orderLineBL.convertQtyEnteredToPriceUOM(ol).toBigDecimal();
			ol.setQtyEnteredInPriceUOM(qtyEnteredInPriceUOM);
```

background
* qtyOrdered is in the product's stock unit-of-measurement ("UOM", column `M_Product.C_UOM_ID`)
* QtyEnteredInPriceUOM needs to be in the price UOM (`C_OrderLine.Price_UOM_ID`)

Reply:
PS: please make sure to look over your PR before devtesting etc, as suggested by the R&I template. 
In that case you might have noticed the discrepancy in column-names between `qtyOrdered` / `convertQtyEnteredToStockUOM` on one side and `qtyEnteredInPriceUOM` on the oter side.